Content.Client/MachineLinking/UI/SignalPortSelectorMenu.xaml

```diff
@@ -1,12 +1,14 @@
-<DefaultWindow xmlns="https://spacestation14.io" Title="{Loc signal-port-selector-menu-title}" MinSize="400 200">
```

Comment:
Instead of having a set or min size, the window should automatically determine a reasonable sized based on its own contents, so that the size is reasonable regardless of the names of the entities being linked.

Reply:
This has frustrated me for a while with several windows, but I think I now understand how to automatically get this behaviour? Basically:

Add something like
```cs
protected override Vector2 ContentsMinimumSize => Contents.DesiredSize + (75,50);
```
to `SignalPortSelectorMenu`, and then add
```cs
Contents.InvalidateMeasure();
Contents.Measure(Vector2.Infinity);
```
to the end of the `UpdateState()` function.